### DISCUSSION TITLE
Throughput Test ：QUIC is only one-fifth of tcp

### DISCUSSION THREAD

**Discussion by noah-hwang** [microsoft/msquic#3753] [Q&A] [answered]
hello!
I tested the throughput with the official secnetperf tool
Results are as follows

QUIC: 
 secnetperf.exe -test:tput -target:127.0.0.1 -upload:10000 -timed:1 -stats:1

[conn][00000218C64C7C30] STATS: EcnCapable=0 RTT=214668 us SendTotalPackets=117660 SendSuspectedLostPackets=0 SendSpuriousLostPackets=0 SendCongestionCount=0 SendEcnCongestionCount=0 RecvTotalPackets=2295 RecvReorderedPackets=0 RecvDroppedPackets=0 RecvDuplicatePackets=0 RecvDecryptionFailures=0
Result: 168624128 bytes @ 120831 kbps (11164.213 ms).
------------------------------------------------------------------
TCP/TLS:
secnetperf.exe -test:tput -target:127.0.0.1 -upload:10000 -timed:1 -stats:1 -tcp:1

Started!

Result: 10124263424 bytes @ 8099177 kbps (10000.288 ms).
-------------------------------------------------------------------
Test results show  that: 
QUIC is only one-fifth of tcp, the results do not seem satisfactory
Is this result normal?
Or is there something wrong with my testing process?
thanks！

**Comment by nibanks** [comment]
Could you [collect](https://github.com/microsoft/msquic/blob/main/docs/Diagnostics.md#wprexe) `Stacks.Light` and share the ETL for both scenarios, along with the .pdb for msquic and secnetperf that you used? Thanks!

**Comment by noah-hwang** [comment]
It may take a while to gather this information because I am not very familiar with the information you mentioned.
I would like to know the official test results，about QUIC and TCP throughput  comparison, thank you！

**Reply by Wizmann** [reply, marked as answer]
For the MsQuic perf results, please check: [this page].

May I ask which OS you are using when running the tests? Using WS2022 or Win11 could bring better performance numbers comparing to older Windows versions according to my experience.

[this page]: https://microsoft.github.io/msquic/

**Reply by noah-hwang** [reply]
I'm using Win 10，Thanks your share
